src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +28,7 @@
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("Successful fetched data from DB for userId: {}",userId);
```

Comment:
The log message indicates successful data fetching from the database, but the method currently returns hardcoded data. Ensure this log message aligns with the actual implementation when integrating with a database.